packages/@aws-cdk/aws-apprunner/test/observability-configuration.test.ts

```diff
@@ -0,0 +1,47 @@
+import { Template } from '@aws-cdk/assertions';
+import * as cdk from '@aws-cdk/core';
+import { ObservabilityConfiguration } from '../lib';
+
+test('create an observability configuration', () => {
```

Comment:
I've asked for a lot of changes so I won't get specific yet but just a forewarning, I'm going to ask for much higher test coverage.